.semaphore/pre-release.yml

```diff
@@ -23,7 +23,7 @@ blocks:
         - name: "Publish to NPM"
           commands:
             - nvm use
-            - npx lerna bootstrap
             - yarn
-            - yarn lerna version prerelease --yes
-            - yarn lerna publish from-package --yes --dist-tag next
+            - test -f .changeset/pre.json || yarn changeset pre enter next
```

Comment:
what is this doing?

Reply:
Just checking if the `pre.json` already there, if it's not there we need to run `yarn changeset pre enter next` to create that file and enter the `pre-release` mode (they called)

If we run `yarn changeset pre enter next` with the file exist, there'll be errors, so I need to check first

Similar with what we need to do in `release.yml`, if the `pre.json` file exist we need run `yarn changeset pre exit`